corehq/feature_previews.py

```diff
@@ -127,3 +127,12 @@ def enable_callcenter(domain_name, checked):
     ),
     privilege=LOOKUP_TABLES,
 )
+
+
+SHARE_CASE_TILE_CONTEXT = FeaturePreview(
```

Comment:
In fact since this is basically an extension to the case tiles feature I don't think it needs a separate toggle at all and can just piggy back on the `CASE_LIST_TILE` toggle.